### PR TITLE
bump to use 0.10 Kafka Client

### LINKED ISSUE #70: Support Kafka 0.10
Adding an enhancement request for supporting Kafka v0.10.

### PR DESCRIPTION
**Pull request by talevy**
bump to use new 0.10 Kafka Client.

*Warning*: this is not backwards compatible

Fixes #70.

### CODE REVIEW

File: logstash-output-kafka.gemspec [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
 
   s.name            = 'logstash-output-kafka'
-  s.version         = '4.0.0'
+  s.version         = '5.0.0'
```
Comment (suyograo) [inline logstash-output-kafka.gemspec]: Why? 4.0.0 is what we'll ship with 5.0 correct? Is the 0.10 producer not compatible with 0.9 broker?

**Comment by suyograo** [comment]
> Warning: this is not backwards compatible

It breaks compatibility with what exactly? 0.9 broker?

**Comment by suyograo** [comment]
LGTM. @talevy and I talked offline about bumping major version for both input and output. I was confused because we hadn't bumped it in input. https://github.com/logstash-plugins/logstash-input-kafka/pull/94/files

**Comment by thoughtpoet** [comment]
Super awesome. Thank you!

**Pull request merged**